### PR TITLE
Update hotkey handling

### PR DESCRIPTION
**Pull request by RodrigoHolztrattner**
Implements a method that will expose all current loaded hotkeys for the obs.
Removes the object ID parameter from the ProcessHotkeyStatus().
Move these hotkey methods to the API class.

### CODE REVIEW

File: obs-studio-client/source/nodeobs_api.cpp [hunk 1]
```diff
@@ -34,10 +34,12 @@ void api::OBS_API_initAPI(const v8::FunctionCallbackInfo<v8::Value>& args)
 			return;
 		}
 	}
-
+
```
Comment (EddyGharbi) [inline obs-studio-client/source/nodeobs_api.cpp]: Can you remove that extra line and space?

File: obs-studio-client/source/nodeobs_api.cpp [hunk 2]
```diff
@@ -34,10 +34,12 @@ void api::OBS_API_initAPI(const v8::FunctionCallbackInfo<v8::Value>& args)
 			return;
 		}
 	}
-
+	
 	args.GetReturnValue().Set(v8::Number::New(args.GetIsolate(), response[1].value_union.i32));
 }
 
+#include "isource.hpp"
```
Comment (EddyGharbi) [inline obs-studio-client/source/nodeobs_api.cpp]: I would keep putting includes at the top of the files, easier to read.

File: obs-studio-server/source/nodeobs_api.cpp [hunk 3]
```diff
@@ -622,6 +627,129 @@ void OBS_API::OBS_API_getPerformanceStatistics(
...
+    void*                          data,
+    const int64_t                  id,
+    const std::vector<ipc::value>& args,
+    std::vector<ipc::value>&       rval)
+{
+	obs_hotkey_id hotkeyId = args[0].value_union.ui64;
+	uint64_t      press    = args[1].value_union.i32;
+
+	// TODO: Check if the hotkey ID is valid
+	obs_hotkey_enable_callback_rerouting(true);
```
Comment (EddyGharbi) [inline obs-studio-server/source/nodeobs_api.cpp]: Do we need to set that to false at some point or obs does it when the callback is done with its work?

**Review by EddyGharbi — CHANGES_REQUESTED** [review]

File: obs-studio-client/source/nodeobs_api.cpp (hunk 1 above)
Comment (RodrigoHolztrattner) [reply obs-studio-client/source/nodeobs_api.cpp]: Done

File: obs-studio-client/source/nodeobs_api.cpp (hunk 2 above)
Comment (RodrigoHolztrattner) [reply obs-studio-client/source/nodeobs_api.cpp]: Removed it (unnecessary include), I was probably doing some testing and forgot to put it at the include section.

File: obs-studio-server/source/nodeobs_api.cpp (hunk 3 above)
Comment (RodrigoHolztrattner) [reply obs-studio-server/source/nodeobs_api.cpp]: Moved it to the obs initialization, we just need to enable it one time.

**Review by EddyGharbi — APPROVED** [review]

**Pull request merged**